src/tools/r2rdump/R2RSection.cs

```diff
@@ -46,10 +46,15 @@ public R2RSection(SectionType type, int rva, int size)
         public override string ToString()
         {
             StringBuilder sb = new StringBuilder();
-            sb.AppendFormat($"Type:  {Enum.GetName(typeof(SectionType), Type)} ({Type:D})\n");
-            sb.AppendFormat($"RelativeVirtualAddress: 0x{RelativeVirtualAddress:X8}\n");
-            sb.AppendFormat($"Size: {Size} bytes\n");
+            sb.AppendLine($"Type:  {Enum.GetName(typeof(SectionType), Type)} ({Type:D})");
+            sb.AppendLine($"RelativeVirtualAddress: 0x{RelativeVirtualAddress:X8}");
+            sb.AppendLine($"Size: {Size} bytes");
             return sb.ToString();
         }
+
+        public string DumpBytes(R2RReader r2r)
```

Comment:
I'm not sure that you need this DumpBytes call. I think it might be better for the caller to instead call DumpBytes on the R2RReader object and supply it with the R2RSection's `RelativeVirtualAddress` and `Size`.

Reply:
The same comment goes for the other DumpBytes methods in the R2RMethod, R2RRuntimeFunction, and R2RHeader.